src/message/html/HtmlNodeTag.js

```diff
@@ -37,15 +37,18 @@ const specialTags = {
   'td': HtmlTagDiv,
 };
 
+const stylesFromClassNames = (classNames = '', styleObj) =>
+  classNames.split(' ').map(className => styleObj[className]);
```

Comment:
This will not throw an exception, but work exactly as you want, the result will be [123, undefined] for example - 123 being the index of a style inside a StyleSheet and undefined will be ignored by RN.